src/main/java/pl/arturzgodka/controllers/SkillsController.java

```diff
@@ -0,0 +1,94 @@
+            "breath-of-heaven", "dashing-strike", "crippling-wave", "wave-of-light", "exploding-palm", "cyclone-strike", "way-of-the-hundred-fists",
+            "serenity", "sevensided-strike", "mantra-of-salvation", "sweeping-wind", "mantra-of-retribution", "inner-sanctuary", "mystic-ally",
+            "mantra-of-healing", "mantra-of-conviction", "epiphany");
+    private final List<String> necromancerSkills = Arrays.asList("bone-spikes", "bone-spear", "grim-scythe", "corpse-explosion", "skeletal-mage", "corpse-lance",
+            "command-skeletons", "siphon-blood", "death-nova", "command-golem", "decrepify", "devour", "leech", "bone-armor", "army-of-the-dead", "frailty",
+            "revive", "bone-spirit", "blood-rush", "land-of-the-dead", "simulacrum");
+    private final List<String> witchDoctorSkills = Arrays.asList("poison-dart", "grasp-of-the-dead", "corpse-spiders", "summon-zombie-dogs", "firebats", "horrify",
+            "soul-harvest", "plague-of-toads", "haunt", "sacrifice", "zombie-charger", "spirit-walk", "spirit-barrage", "gargantuan", "locust-swarm", "firebomb",
+            "hex", "acid-cloud", "mass-confusion", "big-bad-voodoo", "wall-of-death", "fetish-army", "piranhas");
+    private final List<String> wizardSkills = Arrays.asList("magic-missile", "ray-of-frost", "shock-pulse", "frost-nova", "arcane-orb", "diamond-skin",
+            "wave-of-force", "spectral-blade", "arcane-torrent", "energy-twister", "ice-armor", "electrocute", "slow-time", "storm-armor", "explosive-blast",
+            "magic-weapon", "hydra", "disintegrate", "familiar", "teleport", "mirror-image", "meteor", "blizzard", "energy-armor", "archon", "black-hole");
+    private final SkillMapper skillMapper = new SkillMapper();
+    private List<HeroSkillDataModel> barbarianSkillsFetched = skillMapper.fetchSkills("barbarian", barbarianSkills);
+    private List<HeroSkillDataModel> crusaderSkillsFetched = skillMapper.fetchSkills("crusader", crusaderSkills);
+    private List<HeroSkillDataModel> demonHunterSkillsFetched = skillMapper.fetchSkills("demon-hunter", demonHunterSkills);
+    private List<HeroSkillDataModel> monkSkillsFetched = skillMapper.fetchSkills("monk", monkSkills);
+    private List<HeroSkillDataModel> necromancerSkillsFetched = skillMapper.fetchSkills("necromancer", necromancerSkills);
+    private List<HeroSkillDataModel> witchDoctorSkillsFetched = skillMapper.fetchSkills("witch-doctor", witchDoctorSkills);
+    private List<HeroSkillDataModel> wizardSkillsFetched = skillMapper.fetchSkills("wizard", wizardSkills);
+
+    @RequestMapping("/heroClasses.html")
+    public String getHeroesList(Model model) { //model przakzuje aby miec pelnie MVC
+        model.addAttribute("heroClassesList", heroClasses); //model.addStribute podake nazwe zmiennje i skad ma pchodzic wartosc dla tej zmiennej.
+        return "heroClasses";
+    }
+
+    @RequestMapping("/BarbarianSkills.html")
+    public String getBarbarianSkills(Model model) {
```

Comment:
I suggest to do a page dedicated for showing skills and in the model pass the class, so that there are no repetitions. Imagine if you were to do an ecommerce shop where you create seperate html file for each product

Reply:
done, now it is one method to generate skills page for class selected from the list.